README.md

```diff
@@ -1 +1,177 @@
-# Catalog-my-things
\ No newline at end of file
+<a name="readme-top"></a>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Catalog of my things - Ruby capstone <a name="about-project"></a>
+
+> This is the Catalog of my things app initiated in the fifth week of the fourth module in the Microverse program. It is a simple database app to catalog entertainment content (Books, Music, Games).
+
+## Built With
+![](https://img.shields.io/badge/-Ruby-red)
+![](https://img.shields.io/badge/-VSCode-blue)
+![](https://img.shields.io/badge/-Rspec-red)
+- Visual Studio Code
+- Ruby
+- Rspec
+
+
+ ## App ScreenShot
+  ![](Comming Soon)
+ ## Video Preview
+
+ [Video](Comming Soon)
+
```

Comment:
- [ ] Kindly make sure to record a video presentation of the project. The video has to abide by the [following rules](https://github.com/microverseinc/curriculum-ruby/blob/main/group-capstone/catalog_of_my_things.md#:~:text=Once%20you%20have,with%20your%20peers.) given in the requirements.